src/Reveal.Sdk.Dom.Tests/Visualizations/Extensions/Visualizations/GridVisualizationExtensionsFixture.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Reveal.Sdk.Dom.Visualizations;
+using Reveal.Sdk.Dom.Visualizations.Settings;
+using System;
+using System.Collections.Generic;
+using System.Drawing.Printing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations.Extensions.Visualizations
+{
+    public class GridVisualizationExtensionsFixture
+    {
+        [Fact]
+        public void ConfigureSettings_UpdateSettingsField_WithoutConditions()
+        {
+            // Arrange
+            var vs = new GridVisualization();

```

Comment:
This is internal constructor, so is this correct to create it without datasource Item?